src/scss/layouts/_header.scss

```diff
@@ -58,7 +58,7 @@
 
     &:checked ~ .header__list {
       @include absolute($top: 1rem);
-      z-index: calc($zindex1 + 1);
+      z-index: calc($zindex1 + 4);
```

Comment:
Giờ anh mới để ý, đặt tên z-index rõ ràng hơn nhé. Có thể tham khảo trang dev.to í. Nhưng sao z-index nằm ngoài mixin absolute nhỉ?